app/db/models.py

```diff
@@ -305,6 +315,48 @@ class DatasourceExpense(Base, HumanReadablePKMixin, AuditableMixin):
     )
 
 
+class ExchangeRates(Base, HumanReadablePKMixin, AuditableMixin):
```

Comment:
IMO it should inherit from TimestampMixin, who creates, updates this object is always the command under cron schedule, so the *_by will always be null.

Reply:
👍